_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -699,6 +699,69 @@ static mfxStatus SetROI(
     return MFX_ERR_NONE;
 }
 
+mfxStatus SetRoundingOffset(
+    VADisplay    vaDisplay,
+    VAContextID  vaContextEncode,
+    mfxExtAVCRoundingOffset const & roundingOffset,
+    VABufferID & roundingOffsetBuf_id)
+{
+    VAStatus vaSts;
+    VAEncMiscParameterBuffer *misc_param;
+    VAEncMiscParameterCustomRoundingControl *rounding_param;
+
+    if ( roundingOffsetBuf_id != VA_INVALID_ID)
+    {
+        vaDestroyBuffer(vaDisplay, roundingOffsetBuf_id);
```

Comment:
vaDestroyBuffer returns status as well, so you have to check it

Reply:
OK